### PR TITLE
DEV-3396 Fix logic inversion

### PR DESCRIPTION
**Pull request by nedleitch**
I changed the polarity of the `jobIs` method because Intellij was complaining that it was always inverted and ended up causing more problems than the change was worth.

This fixes the logic.

### CODE REVIEW

File: src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java [hunk 1]
```diff
@@ -37,7 +37,7 @@ public ConstantJobCountScheduler(final JobSubmitter jobSubmitter, final Kubernet
     }
 
     private static boolean jobIsNot(Integer status) {
```
Comment (pauldwolfe) [inline src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java:39]: This name doesn't quite match, seems that jobIsNotFailed (is that what 1 means?) + a constant might help with some confusion down the line?

**Review by pauldwolfe — CHANGES_REQUESTED** [review]

File: src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java (hunk 1 above)
Comment (nedleitch) [reply src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java:39]: I use it for other statuses too, it's a funny API. The job has essentially several statuses (active, failed, etc) and then each of those is an Integer or null.

If it's 1 then it didn't fail. If it is null or non-1 it did, or maybe it hasn't gotten to that stage of its lifecycle yet.

Fair point on the constant.

Comment (nedleitch) [reply src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java:39]: Here's how it gets called: 

```
                            if (jobIsNot(job.getStatus().getActive())) {
                                if (jobIsNot(job.getStatus().getFailed())) {
```

File: src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java [hunk 2]
```diff
@@ -54,6 +50,14 @@ public void submit(final PipelineJob job) {
         }
     }
 
+    private boolean jobCompleted(Job job) {
+        return job.getStatus().getConditions().stream().anyMatch(c -> "Complete".equals(c.getType()) && "True".equals(c.getStatus()));
```
Comment (pauldwolfe) [inline src/main/java/com/hartwig/platinum/scheduling/ConstantJobCountScheduler.java:54]: you could remove a bit of duplication here between the condition checks.

**Review by pauldwolfe — CHANGES_REQUESTED** [review]

**Review by pauldwolfe — APPROVED** [review]

**Pull request merged**